CMakeLists.txt

```diff
@@ -72,6 +73,10 @@ else()
   set(CMAKE_CXX_FLAGS " -O3 -g ${CMAKE_CXX_FLAGS}")
 endif()
 
+if(ENABLE_GPROF)
+  set(CMAKE_CXX_FLAGS " -pg ${CMAKE_CXX_FLAGS}")
```

Comment:
gprof may not be available in macOS， you can print some warning

Reply:
Done